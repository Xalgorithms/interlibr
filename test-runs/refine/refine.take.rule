META
  VERSION "0.0.1"
  MAINTAINER "Don Kelly <karfai@gmail.com>";

REQUIRE refine:table0:0.0.1 AS table0;

REFINE table:table0 AS table1
  TAKE first(3);

REFINE table:table0 AS table2
  TAKE nth(1, 3);

REFINE table:table0 AS table3
  TAKE a == 3;
