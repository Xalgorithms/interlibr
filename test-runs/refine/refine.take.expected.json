{
  "tables" : {
    "table1" : [
      { "a" : 1, "b" : 2  },
      { "a" : 2, "b" : 4  },
      { "a" : 3, "b" : 9  }
    ],
    "table2" : [
      { "a" : 2, "b" : 4  },
      { "a" : 3, "b" : 9  },
      { "a" : 4, "b" : 16 }
    ],
    "table3" : [
      { "a" : 3, "b" : 9  }
    ]
  }
}
